root/admin/wikidoc/WikiDocIndex.js

```diff
@@ -0,0 +1,191 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *      /*  ((n   *'.
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {defaults} from 'lodash';
+
+import Layout from '../../layout';
+import Table from '../../components/Table';
+import {withCatalystContext} from '../../context';
+
+import type {WikiDocT} from './types';
+
+type PropsT = {
+  +$c: CatalystContextT,
+  +pages: $ReadOnlyArray<WikiDocT>,
+  +updates_required: boolean,
+  +wiki_server: string,
+  +wiki_unreachable: boolean,
+};
+
+const WikiDocTable = withCatalystContext(({
+  $c,
+  pages,
+  updates_required,
+  wiki_server,
+}: PropsT) => {
+  const columns = React.useMemo(
+    () => {
+      const nameColumn = {
+        Cell: ({cell: {value}}) => (
+          <a href={'/doc/' + encodeURIComponent(value)}>{value}</a>
+        ),
+        Header: N_l('Page name'),
+        accessor: 'id',
+        cellProps: {className: 'title'},
+        id: 'name',
+      };
+      const transcludedVersionColumn = {
+        Header: N_l('Transcluded version'),
+        accessor: 'version',
+        cellProps: defaults(
+          {className: 'c transcluded-version'},
+          (updates_required && $c.user?.is_wiki_transcluder
+            ? {style: {textAlign: 'right'}}
+            : {}
+          ),
+        ),
+        headerProps: {className: 'c'},
+        id: 'transcluded-version',
+      };
+      const wikiVersionColumn = {
+        Cell: ({row: {original}}) => (
+          <>
+            {original.wiki_version === original.version ? null : (
+              <>
+                <span
+                  className="wiki-version"
+                  style={{color: 'red'}}
+                >
+                  {original.wiki_version === 0
+                    ? l('Error!')
+                    : original.wiki_version}
+                </span>
+                {original.wiki_version ? (
+                  <>
+                    {' '}
+                    <a
+                      href={'//' + wiki_server +
+                            '/' + encodeURIComponent(original.id) +
+                            '?diff=' + original.wiki_version +
+                            '&oldid=' + original.version}
+                    >
+                      {l('diff')}
```

Comment:
This was bracketed in TT (and it probably makes sense to keep it that way)

Reply:
Good catch, fixed in https://github.com/metabrainz/musicbrainz-server/pull/1461/commits/0b7e06843a742fb87934f5f9bb0044ee5b2d330f